scripts/syschdemd.nix

```diff
@@ -0,0 +1,51 @@
+{ runCommand
+, makeWrapper
+, lib
+, coreutils
+, daemonize
+, glibc
+, gnugrep
+, systemd
+, util-linux
+, defaultUser
+, automountPath
+,
+}:
+let
+  writeShellScript =
```

Comment:
This shadows a function in nixpkgs https://github.com/NixOS/nixpkgs/blob/master/pkgs/build-support/trivial-builders.nix#L213